examples/nextjs/src/components/my-page.js

```diff
@@ -71,7 +71,7 @@ export function MyPage({ pageAsset, nav }) {
                     config={{ onReload: refresh, pathname }}
                 />
             </main>
-            {pageAsset.layout.footer && <Footer />}
+            {pageAsset.layout.footer && <Footer pageAsset={pageAsset} />}
```

Comment:
WHAT! Why?

Reply:
Yeah I hate prop drilling but I had to in order to make it work at the moment. We can brainstorm to see if anyone has an idea of how to approach this in a better way